### ISSUE TITLE
Adding a Few Words About What is Being Shared

### ISSUE THREAD

**Issue by mehmetakiftutuncu** [SalyangozMe/feedback#10]
I think it would be cool to be able to say a little something about the link being shared before actually sharing it. :)

**Comment by vigo** [comment]
Exactly. I want to have an idea before I click the url. Few words would be kool.
